src/sql/base/browser/ui/uiControlBuilder.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Button } from 'sql/base/browser/ui/button/button';
+import views = require('views');
```

Comment:
I think our style is to use `import * as views from 'views'` instead of `require`, though I don't think it actually makes a functional difference